src/main/resources/Diagram/MigrationScript.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<xwikidoc version="1.3" reference="Diagram.MigrationScript" locale="">
+  <web>Diagram</web>
+  <name>MigrationScript</name>
+  <language/>
+  <defaultLanguage/>
+  <translation>0</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <creationDate>1549541592000</creationDate>
+  <parent>xwiki:Diagram.WebHome</parent>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <date>1549541642000</date>
+  <contentUpdateDate>1549541592000</contentUpdateDate>
+  <version>1.1</version>
+  <title>MigrationScript</title>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>true</hidden>
+  <content>{{info}}
+This page contains a script for migrating diagram pages after an upgrade of the diagram application to the pro version of the application. It will update the paths containing the version of the draw.io library with the version currently used by the application.
+
+{{html}}
+  &lt;form action="" method="post"&gt;
+  &lt;input class="btn btn-default" type="submit" name="preview" value="Preview"/&gt;
+  &lt;input class="btn btn-default" type="submit" name="execute" value="Execute"/&gt;
+  &lt;/form&gt;
+{{/html}}
+
+{{/info}}
+
+{{groovy}}
+  
+
+String getDependencyVersion(String extensionId, String dependencyId) {

```

Comment:
You can't have multiple versions of the same extension installed at the same time on the same namespace. So there's only one version of ``org.xwiki.contrib:draw.io`` installed and you can ask for it directly with:

    getInstalledExtension('org.xwiki.contrib:draw.io', "wiki:$xcontext.database")

Reply:
Done this way indeed, thanks.